portfolio/src/main/webapp/comments.html

```diff
@@ -66,6 +66,14 @@ <h2>Comment History</h2>
       <span class="commentparam">
         <button onclick="clearComments()">Clear</button>
       </span>
+      <div id="filterdiv">
+        <select name="filterby" id="filterby" onchange="loadComments()">
```

Comment:
The same way the label "Sort by:" preceded the select element, add a "Filter by:" text so that it makes it clear for the users what the select element is for?